### ISSUE TITLE
README with build instructions

### ISSUE THREAD

**Issue by jschwartzenberg** [stsp/comcom32#36]
This repo needs it :)

**Comment by jschwartzenberg** [comment]
Yeah that needs to be documented :)

Hope distros will adopt DJGPP though, it would help a lot.

**Comment by stsp** [comment]
Could you please send a patch? :)

**Comment by jschwartzenberg** [comment]
Are you aware of any existing packages for non-Ubuntu systems?

**Comment by stsp** [comment]
ftp://ftp.delorie.com/pub/djgpp/rpms/

**Comment by argenkiwi** [comment]
I've been trying to build dosemu2 on Tiny Core Linux. I managed to get it running, but I was seeing the error @jschwartzenberg mentioned above. I had to download the binaries for DJGPP and set some environment variables manually to be able to run `make`. After that, the only way I was able to execute dosemu with it was by copying `command.com` into `drive_c` manually. Is that expected?

**Comment by stsp** [comment]
No.
You should do `sudo make install`
and dosemu2 should pick that up.

**Comment by argenkiwi** [comment]
Thanks. That seems to work.

I was trying to build a tcz extension for Tiny Core which entails running `sudo make DESTDIR=tmp/[a directory] install` and then packaging it using `mksquashfs`  and then loading the extension but that wasn't working.

I now get an error when dosemu starts: "Cannot open D:\FDPPAUTO.BAT" and then I cannot execute any other `.BAT` or `.EXE` files. Any clues if it is related to this library? I am wondering if there is some permissions issue or if it is due to running on Tiny Core (32 bit). I had to also build dosemu2 and fdpp from source, but I did not notice anything strange when I built them.
